drivers/AnalogIn.h

```diff
@@ -131,6 +160,9 @@ class AnalogIn {
 
     analogin_t _adc;
     static SingletonPtr<PlatformMutex> _mutex;
+
+    float vref;
```

Comment:
Members should be prefixed with an _
```suggestion
    float _vref;
```

Reply:
Renamed member as suggested.